ssh/ssh_tunnel.py

```diff
@@ -54,7 +54,7 @@ def __init__(self, ssh_user, ssh_key_path, host, port=22):
         check_call(start_tunnel)
         logger.debug('SSH Tunnel established!')
 
-    def remote_cmd(self, cmd, timeout=None):
+    def remote_cmd(self, cmd, raw_output=False, timeout=None):
```

Comment:
I agree, this formatting is a holdover from trying to parse information retrieved with remote_cmd through the azure load balancer. For some reason, even though you are on *nix, because you go through the azure LB everything has windows returns. :( Anyway, I just need to remember to account for that when I plug in azure.